recipes/starshot_blog/recipe.yml

```
@@ -7,6 +7,7 @@ recipes:
   - starshot_content_type_base
   - starshot_image_media_type
   - core/recipes/tags_taxonomy
+  - recipes/notify_new_comments
```

This isn't going to do what you think it will. The correct way to refer to this would be:

```suggestion
  - notify_new_comments
```
The path-like things are only used for core, because core is special-cased.